lib/LaTeXML/Post/MathML.pm

```diff
@@ -768,10 +710,48 @@ sub stylizeContent {
     $text    = $u_text;
     $variant = ($plane1hack && ($variant ne $u_variant) && ($variant =~ /^bold/)
       ? 'bold' : undef); }                       # Possibly keep variant bold
-                                                 # Other attributes that should be copied?
-  my $istoken = $tag =~ /^m:(?:mi|mo|mn)$/;      # mrow?
-  my $href    = $istoken && ($iselement ? $item->getAttribute('href')  : $attr{href});
-  my $title   = $istoken && ($iselement ? $item->getAttribute('title') : $attr{title});
+                                                 # Use class (css) to patchup some weak translations
+  if    (!$font) { }
+  elsif ($font =~ /caligraphic/) {
+    # Note that this is unlikely to have effect when plane1 chars are used!
+    $class = ($class ? $class . ' ' : '') . 'ltx_font_mathcaligraphic'; }
+  elsif ($font =~ /script/) {
+    $class = ($class ? $class . ' ' : '') . 'ltx_font_mathscript'; }
+  elsif (($font =~ /fraktur/) && ($text =~ /^[\+\-\d\.]*$/)) {    # fraktur number?
+    $class = ($class ? $class . ' ' : '') . 'ltx_font_oldstyle'; }
+  elsif ($font =~ /smallcaps/) {
+    $class = ($class ? $class . ' ' : '') . 'ltx_font_smallcaps'; }
+  if ($opacity) {
+    $cssstyle = ($cssstyle ? $cssstyle . ';' : '') . "opacity:$opacity"; }
+
+  # Now, look up any OperatorDictionary properties
+  my %props = ($tag eq 'm:mo' ? opdict_lookup($text, $role) : ());
+
+  # Resolve stretch & size
+  $stretchy = undef if ($tag ne 'm:mo');                          # Only allowed on m:mo!
+  $size     = undef if $stretchy;                                 # Ignore size, if we're stretching.
+  my $stretchyhack = undef;
+  if ($text =~ /^[\x{2061}\x{2062}\x{2063}]*$/) {    # invisible get no size or stretchiness
+    $stretchy = $size = undef; }
+  if ($size) {
+    if ($size eq ($LaTeXML::MathML::SIZE || 'text')) {    # If default size, no need to mention.
+      $size = undef; }
+    # If requested relative size, and in script or scriptscript, we'll need to adjust the size
+    elsif (($size =~ /%$/) && ($LaTeXML::MathML::STYLE and $LaTeXML::MathML::STYLE =~ /script/)) {
+      my $req = $size;                               $req =~ s/%$//;
+      my $ex  = $stylesize{$LaTeXML::MathML::STYLE}; $ex  =~ s/%$//;
+      $size = int(100 * $req / $ex) . '%'; }
```

Comment:
it would help to add a guard that ensures `$ex` is never zero or undefined here, or we may be at risk for a perl die.

Reply:
Yeah, good idea.